src/main/resources/changesets/changelog-consent-105.0.xml

```diff
@@ -0,0 +1,11 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="105.0" author="grushton">
+        <sql>
+            ALTER TABLE user_property
+            ADD CONSTRAINT uc_user_property_userid_keyval
+            UNIQUE (userid, propertykey);
+        </sql>
```

Comment:
would there be an advantage/disadvantage to using [`<addUniqueConstraint... />`](https://docs.liquibase.com/change-types/add-unique-constraint.html) here over raw SQL? it looks like it has support for multiple column unique constraints.

Reply:
In theory, there should be no difference. Under the hood, Liquibase turns the xml into sql.